libraries/load-themed-styles/src/index.ts

```diff
@@ -62,6 +63,15 @@ interface IStyleRecord {
 }
 
 /**
+ * object returned from resolveThemableArray function
+ * @styleString:  this string is the processed styles in string
+ * @themable:     this boolean indicates if this style array is themable
+ */
+interface IThemableArrayResolveResult {
+  styleString: string;
+  themable: boolean;
+}
+/**
  * In sync mode, styles are registered as style elements synchronously with loadStyles() call.
```

Comment:
A a newline between the end of the interface and the documentation comment for the enum.